controllers/inventory_controller.go

```diff
@@ -0,0 +1,73 @@
+package controllers
+
+import (
+	"context"
+
+	"github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
```

Comment:
Suggesting to use an alias here. This will make it easier to replace it with a beta import in the future.

```suggestion
	build "github.com/shipwright-io/build/pkg/apis/build/v1alpha1"
```

Reply:
Let's tackle this on #9.